### DISCUSSION TITLE
Matching step on related entity

### DISCUSSION THREAD

**Discussion by radaniba** [marklogic-community/envision#253] [Q&A]
Hi there,

I am wondering if we use the attributes in a given entity that are configured as 'Related entity' in the matching step, currently I have it showing as "Multiple" but there is no way of me selecting what attribute in particular I want to match against 

Any idea ?

"is_structured_for" here refers to an entity in my "Client" entity that is referring to an "Appointment" entity and I would like to match against one of its attributes to build the score for matching

![Screen Shot 2021-05-17 at 3 36 09 pm](https://user-images.githubusercontent.com/1666443/118449831-1a21e000-b726-11eb-812b-570d9998f09c.png)

**Comment by radaniba** [comment]
and I noticed that in the documentation the add Ruleset has a dropdown menu to add mutiple properties, where as in my 5.4.3 installation I have only one button for without that option so I can only map a single property at a time

## From the Documentation

![Screen Shot 2021-05-17 at 3 56 09 pm](https://user-images.githubusercontent.com/1666443/118452388-aaf9bb00-b728-11eb-87f5-c52dc4fba5c6.png)

## From my 5.4.3 instance

![Screen Shot 2021-05-17 at 3 58 54 pm](https://user-images.githubusercontent.com/1666443/118452579-e1373a80-b728-11eb-8094-8993f79dd119.png)

**Comment by wpaven** [comment]
I think this is an issue in the UI.  is_scheduled_for appears to be a relationship, not a property.  is that the case?

**Reply by radaniba** [reply]
No, I should have described the label better, but it is in fact a property, I will cross post to HC related channels to make sure it is a bug or not. Thanks for your support @wpaven

**Reply by wpaven** [reply]
Thanks @radaniba
